### ISSUE TITLE
Allow use of other criteria

### ISSUE THREAD

**Issue by sims1253** [stan-dev/loo#201]
It would be nice if the loo objects weren't hardcoded to use elpd. For a study I wanted to include rmse and R2 and had to build fake loo objects in order to use the loo_compare functionality. Code is [here](https://github.com/sims1253/bayesim/blob/master/R/loo_handler.R).

**Comment by avehtari** [comment]
@sims1253 do you need something else than what is in the PRs mentioned above, or can we close this issue?

**Comment by avehtari** [comment]
We do have rmse, but we're still missing R2 (which is available in brms and rstanarm, but not as plain loo package version)

**Comment by sims1253** [comment]
> ...which is available in brms

At least the brms version does not use psis, which is why I wrote the version from my pr.

**Comment by avehtari** [comment]
brms `loo_R2` does use psis based on the code https://github.com/paul-buerkner/brms/blob/d1a050803670f4f4647324ce9a84780591a40692/R/loo_predict.R#L202

**Comment by sims1253** [comment]
True. Sorry, I'm currently on vacation and only have my phone and only found the Bayes r2 function. 
Maybe it was about the pointwise estimate for the SE then that we built the new version. I don't quite remember to be honest.
